src/utils/stringToBoolean.js

```diff
@@ -0,0 +1,12 @@
+export const stringToBoolean = string => {
+  switch (string.toLowerCase().trim()) {
+    case 'true':
+      return true
+    case 'false':
+    case '0':
+    case null:
+      return false
+    default:
+      return Boolean(string)
```

Comment:
This gives `false` if and only if `string` is one of: `undefined`, `null`, `false`, `0`, `-0`, `NaN`, or `''`.
For any other value in JS (including the values: `'false'`, `{}`, & `[]`) it gives `true`.

I'm not sure I understand what the function `stringToBoolean` should do…